owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt

```diff
@@ -56,10 +56,36 @@ class ReleaseNotesViewModel(
                 type = ReleaseNoteType.ENHANCEMENT,
             ),
             ReleaseNote(
-                title = R.string.release_notes_4_3_0_title_3,
-                subtitle = R.string.release_notes_4_3_0_subtitle_3,
+                title = R.string.release_notes_4_3_0_title_new_ui_manage_accounts,
+                subtitle = R.string.release_notes_4_3_0_subtitle_new_ui_manage_accounts,
+                type = ReleaseNoteType.ENHANCEMENT
+            ),
```

Comment:
I think this one is the main visual change of this release, which is the kind of changes the users value the most

Reply:
Ordered at the top